src/octodns_netbox_dns/__init__.py

```diff
@@ -193,7 +196,7 @@ def _format_rdata(self, rdata: dns.rdata.Rdata, raw_value: str) -> str | dict[st
                 }
 
             case "SPF" | "TXT":
-                value = raw_value.replace(";", r"\;")
+                value = raw_value.replace(";", "\\;")
```

Comment:
I would like to keep the raw string as its clearer what the replacement exactly is.

Reply:
The reason I did that is because in lines 325 and 402, I needed to use a single backslash and ```r"\"``` isn't valid, so I used ```"\\"```.  Since I had that one in that syntax, I made all the others the same.  It's your call if you want to keep just that one as raw, make everything else except the singular ```"\\"``` raw or leave it the way I put it.

Unless you know a different syntax that gives you something equivalent to ```r"\"```.

Reply:
Ah i get it. Then its ok and we keep it all the same (your approach)